### ISSUE TITLE
[userContent.css] missing end-comment (*/) in "custom_scrollbar_appearance.css" section (v4.2.8 lines 203-212)

### ISSUE THREAD

**Issue by cbaumer0628** [Aris-t2/CustomCSSforFx#539]
1. Does the issue occur without any files inside /chrome/ folder?
no

2. Issue description, userChrome.css/userContent.css 'settings' the issue occurs with and steps to reproduce the issue:
userContent.css missing end-comment (`*/`) in "custom_scrollbar_appearance.css" section (v4.2.8 lines 203-212)
before "`/* @import ...`" line.

3. Does the issue occur with the most recent version of CustomCSSforFx (release/test/dev build)?
yes, v4.2.8

4. Screenshots showing the issue (drag & drop images into this post):
userContent.css (v4.2.8 lines 203-212):
```
/* ------------------------------------------
   CUSTOM SCROLLBARS VARIABLES - EXPERIMENTAL 
   ------------------------------------------
   
   [!] Might not work on every page! This issue is for Mozilla to solve.
   [!] Not compatible with custom scrollbar JavaScript userChrome scripts.
   [!] Custom size values only compatible with 'web content' scrollbars at the moment.
   [!] More info inside 'custom_scrollbar_appearance.css' file.

/* @import "./config/custom_scrollbar_appearance.css"; /**/
```

5. System information
- OS & OS version: Windows 7 Pro SP1 x64
- DPI / HiDPI (e.g. 100%, 125%...): 100%
- Firefox version: 108.0.1
- Firefox theme (e.g. default, build in light/dark, external theme + link): light 1.2
- Is 'about:config > toolkit.legacyUserProfileCustomizations.stylesheets' set to 'true': yes
- Is 'about:config > layout.css.moz-box-flexbox-emulation.enabled' set to 'false' (Fx 108+): yes

**Comment by Aris-t2** [comment]
Thanks for the note.

**Comment by Aris-t2** [comment]
Fixed in [v4.2.9 release](https://github.com/Aris-t2/CustomCSSforFx/releases/).
